### DISCUSSION TITLE
Why pulse_meter is better of pulse_counter

### DISCUSSION THREAD

**Discussion by array81** [huizebruin/s0tool#134] [General] [answered]
I noticed that in "recent" times the sensor was changed from pulse_counter to pulse_meter in the water meter.
Why?
I use an NPN sensor on my water meter, the meter has a metal pointer. At each turn I count a pulse and therefore a liter. Wouldn't it be better to use the pulse_counter sensor?

**Comment by huizebruin** [comment, marked as answer]
what i heard the pulse_meter gets more updates on the back code on esphome and counts better.
and less fake pulses.
